app/cli/cmd/policy_develop_init.go

```diff
@@ -0,0 +1,79 @@
+//
+// Copyright 2025 The Chainloop Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/chainloop-dev/chainloop/app/cli/internal/action"
+	"github.com/spf13/cobra"
+)
+
+func newPolicyDevelopInitCmd() *cobra.Command {
+	var (
+		force       bool
+		embedded    bool
+		name        string
+		description string
+	)
+
+	cmd := &cobra.Command{
+		Use:   "init [directory]",
+		Short: "Initialize a new policy",
+		Long: `Initialize a new policy by creating template policy files in the specified directory.
+By default, it creates chainloop-policy.yaml and chainloop-policy.rego files.`,
+		Example: `  
+  # Initialize in current directory with separate files
+  chainloop policy develop init
+
+  # Initialize in specific directory with embedded format
+  chainloop policy develop init ./policies --embedded`,
```

Comment:
one thing that we do not do in any command in chainloop is to use arguments, we always use flags. The reason being that they are more flexible and easier to change in the future.